packages/mobile/src/store/store.ts

```diff
@@ -10,11 +10,19 @@ const extraArgument = {
 const store = configureStore({
   reducer: rootReducer,
   middleware: (getDefaultMiddleware) => {
-    return getDefaultMiddleware({
+    const defaultMiddleware = getDefaultMiddleware({
       thunk: {
         extraArgument,
       },
     });
+
+    /* eslint-disable */
+    if (__DEV__) {
```

Comment:
This will disable entire code below that statement. I'd go with more precise 
```suggestion
    if (__DEV__) {
    // eslint-disable-next-line
```

Reply:
corrected